src/components/Header/BaseHeader.jsx

```diff
@@ -89,6 +90,8 @@ BaseHeader.displayName = "BaseHeader";
 BaseHeader.propTypes = {
   /** The application name or title that appears in the header */
   title: PropTypes.string.isRequired,
+  /** The href to use for the skip to content link */
+  contentHref: PropTypes.string.isRequired,
```

Comment:
`contentHref` shouldn't be required. It should default to `#main-content`. That will make it more backwards compatible.

Reply:
Although I do get your point that by forcing users to explicitly set it, it makes them more aware of it and perhaps more likely to forget to set `<main id>`.

I'd welcome other thoughts on this.

@johglove, @pcberg?

Reply:
I'm less concerned about backwards compatibility in beta, especially because we don't enforce the ID `#main-content` on the `main` tag and don't document anywhere that it's what the header expects. This change will only affect users who are upgrading within the beta and I think making it required and documenting it for new users provides more value.

Reply:
Looks like in rivet-react v1 it was defaulted https://github.com/indiana-university/rivet-react/blob/96b2a2d343932f4dc2870c87fdea77cb6f06daa7/src/components/Header/Header.tsx#L33 and we seemed to have noticed for the projects where we use it on our team.

Having said that, if it was me I would not have noticed so an isRequired would have made me aware. I'm in favor of that approach. But I can be convinced either way.